cloud_functions/climateiq_export_to_aws_cf/main.py

```diff
@@ -0,0 +1,72 @@
+from datetime import datetime
+import os
+
+import boto3
+import flask
+import functions_framework
+from google.cloud import firestore, storage
+
+# GCS bucket name, where merged prediction outputs are stored.
+GCS_BUCKET_NAME = (
+    os.environ.get("BUCKET_PREFIX", "") + "climateiq-spatialized-merged-predictions"
+)
+# AWS bucket name, where to copy files to.
+S3_BUCKET_NAME = "climateiq-data-delivery"
+
+# IDs for Firestore.
```

Comment:
lets use secret manager to store the API keys: https://cloud.google.com/functions/docs/configuring/secrets#:~:text=You%20can%20use%20Secret%20Manager,secrets%20stored%20in%20Secret%20Manager.